src/main/descriptors/src.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This would be the only file in the top-level `src` directory, which would make that misleading to visitors.

I'd suggest to put the file, for the time being, inside `sdks/java` directory. The clear downside is that this is not really Java SDK-specific. However, we have `checkstyle` there right now, which has similar scope.